src/AzureDevOpsPolicyConfigurator/Logic/PolicyExecuterBase.cs

```diff
@@ -29,18 +22,25 @@ internal abstract class PolicyExecuterBase : ILogicExecuter<ExecuterSettings>
         /// <param name="serializer">JsonSerialzer</param>
         /// <param name="reader">FileReader</param>
         /// <param name="connectionProvider">Connection provider</param>
-        public PolicyExecuterBase(IJsonSerializer serializer, IFileReader reader, IConnectionProvider connectionProvider)
+        /// <param name="logger">logger</param>
```

Comment:
The comment for `logger` can be removed as `logger` won't provide additional help in terms of describing the param.